src/pages/index.jsx

```diff
@@ -434,45 +436,51 @@ export default function Home({ articles }) {
             <div className="mx-auto max-w-2xl lg:max-w-none">
               <div className="text-center">
                 <h2 className="text-3xl font-bold tracking-tight text-white sm:text-4xl">
-                  Some of our stats
+                  Sponsors!
                 </h2>
                 <p className="mt-4 text-lg leading-8 text-gray-300">
-                  Lorem ipsum dolor sit amet consect adipisicing possimus.
+                  Robyn is powered by these amazing people
                 </p>
               </div>
```

Comment:
Let's keep the stats here and move the sponsors to the `/community` page please?